boot/zephyr/main.c

```diff
@@ -73,7 +81,7 @@ void main(void)
 	SYS_LOG_INF("Setting vector table to %p", vt);
 
 	/* Not all targets set the VTOR, so just set it. */
-	_scs_relocate_vector_table((void *) vt);
+	relocate_vector_table((void *) vt);
```

Comment:
The call to _scs_relocate_vector_table() was just a hack I added, and kind of accidentally committed.